microphone/src/main.c

```diff
@@ -110,21 +117,21 @@ int main(int argc, char *argv[]){
 		printf("\nYour microphone is %s\n\n", sceAudioInGetStatus(1)?"disabled.": "enabled. ");
 		printf("\nSimple VU meter: (sensitivity = %3d)\n\n\n", sensitivity);		
 		for (i = 0; i < MAX_VU; i++){
-			psvDebugScreenCoordX = 56;
+			coordX = 56;
 			if (i < average){
 				if (MAX_VU/2 > i)
-					psvDebugScreenSetBgColor(COLOR_GREEN);
+					psvDebugScreenSetBgColor(0xFF00FF00);
```

Comment:
Why?

Reply:
I like that, but @yne removed in #38 and explained [reason](https://github.com/vitasdk/samples/pull/38#discussion_r153209221)

maybe really need this macro, we should revert psvDebugScreenSet(Bg/Fg) and color macros in another pr :)

Reply:
Maybe the colors should be changed to the CSI sequence thing ?
printf("\e[....m");

Reply:
yes, we can. just easy/non-easy problem :>